### PR TITLE
New ebuild tarantool-c-9999.ebuild. First rel.

### PR DESCRIPTION
**Pull request by D-Y-V**

### CODE REVIEW

File: dev-libs/tarantool-c/tarantool-c-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+if [[ ${PV} = 9999* ]]; then
+        GIT_ECLASS="git-r3"
+fi
+
+inherit eutils cmake-utils $GIT_ECLASS
+
+DESCRIPTION="Tarantool-c connector. Very fast framework, originaly developed by (C)Mail.ru"
+HOMEPAGE="http://tarantool.github.io/tarantool-c/index.html"
+LICENSE="BSD-2 clause"
```
Comment (Totktonada) [inline dev-libs/tarantool-c/tarantool-c-9999.ebuild]: There is `BSD-2` in `/usr/portage/licenses`, so I guess the format should be `LICENSE="BSD-2"`.

Comment (D-Y-V) [reply dev-libs/tarantool-c/tarantool-c-9999.ebuild]: Ok, changed .

Comment (Totktonada) [reply dev-libs/tarantool-c/tarantool-c-9999.ebuild]: Should be `BSD-2`. Anyway, I'm going to change more things. Will push soon.

**Comment by Totktonada** [comment]
Pushed.

**Pull request closed without merging**